pkgs/applications/office/wordgrinder/default.nix

```diff
@@ -41,8 +41,6 @@ stdenv.mkDerivation rec {
   # Binaries look for LuaFileSystem library (lfs.so) at runtime
   postInstall = ''
     wrapProgram $out/bin/wordgrinder --set LUA_CPATH "${lua52Packages.luafilesystem}/lib/lua/5.2/lfs.so";
-  '' + stdenv.lib.optionalString stdenv.isLinux ''
-    wrapProgram $out/bin/xwordgrinder --set LUA_CPATH "${lua52Packages.luafilesystem}/lib/lua/5.2/lfs.so";
```

Comment:
With the suggested change above, it should build with X support again.

Reply:
with the suggested change:

```$ nix-build pkgs/top-level --arg localSystem '{ system = builtins.currentSystem; }' --arg crossSystem null -A wordgrinder --show-trace
error: while evaluating the attribute 'preBuild' of the derivation 'wordgrinder-0.7.2' at /pkgs/applications/office/wordgrinder/default.nix:5:3:
cannot coerce a function to a string, at /pkgs/applications/office/wordgrinder/default.nix:5:3
```

Reply:
Ah, sorry, my bad, corrected the suggestion.